compiler/modules/types/list.bal

```diff
@@ -19,6 +19,10 @@ public type FixedLengthArray record {|
 // The SemTypes in this list are not `never`.
 public type ListMemberTypes [Range[], SemType[]];
 
+public function listAtomicTypeMemberAtInnerVal(ListAtomicType atomic, int i) returns SemType {
```

Comment:
This seems identical to `listAtomicTypeMemberAtInner` (unlike [`mappingAtomicTypeMemberAtInnerVal`](https://github.com/ballerina-platform/nballerina/blob/49cfa7b757a21ff94f0d949925f36d27bfcfc88e/compiler/modules/types/mapping.bal#L18). Is there a reason why we don't need to remove `UNDEF` here, for an arbitrary `ListAtomicType`?

Reply:
Fixed in [83989d4](https://github.com/ballerina-platform/nballerina/pull/1162/commits/83989d418030e0b9f889aafda7a8eba3282ff571)